data/scripts/libsoletta.sym

```diff
@@ -613,6 +613,10 @@ global:
         sol_netctl_service_state_from_str;
         sol_netctl_service_state_to_str;
         sol_netctl_set_radios_offline;
+        sol_netctl_register_agent;
+        sol_netctl_report_error;
+        sol_netctl_request_input;
+        sol_netctl_scan;
```

Comment:
you can't add those to LIBSOLETTA_1 version, you need to open a new block and add them there. Later on you can check with "nm" your symbols and see they'll have a "@@LIBSOLETTA_2".